sos/policies/distros/redhat.py

```diff
@@ -447,7 +447,7 @@ class RedHatAtomicPolicy(RHELPolicy):
 
     containerzed = True
     container_runtime = 'docker'
-    container_image = 'registry.access.redhat.com/rhel7/support-tools'
+    container_image = 'registry.access.redhat.com/rhel9/support-tools'
```

Comment:
This is for RHEL Atomic, which afaik runs on RHEL7 only. And running RHEL9 containers on RHEL7 is not supported.

How far does it make sense to keep this RHEL7-only related class / policy, if RHEL7 does not contain `python3` by default and hence we don't plan any `sos-4.*` release there..? There can be some corner case use cases, maybe..?